pkg/provider/aws/parameterstore/parameterstore.go

```diff
@@ -281,25 +297,49 @@ func (pm *ParameterStore) setExisting(ctx context.Context, existing *ssm.GetPara
 		return nil
 	}
 
-	return pm.setManagedRemoteParameter(ctx, secretRequest, false)
-}
+	err = pm.setManagedRemoteParameter(ctx, secretRequest, []ssmTypes.Tag{}, false)
```

Comment:
Shouldn't this be `tags` here that was constructed previously?

Reply:
Not really. The last parameter is `false` so passing tags will not make any differece, to udpate tags the `override` is required, but is not a recommended way to manage resouce Tags.

I could refactor `setManagedRemoteParameter` function to pass tags

Reply:
Gotcha. Maybe this is fine. :)